VCFv4.3.tex

```diff
@@ -173,30 +173,49 @@ \subsubsection{Individual format field format}
 Possible Types for FORMAT fields are: Integer, Float, Character, and String (this field is otherwise defined precisely as the INFO field).
 
 \subsubsection{Alternative allele field format}
-Symbolic alternate alleles are described as follows:
+Symbolic alternate alleles specified in the ALT field are described as follows:
```

Comment:
I think that this section could use a little clarification or a preamble stating what the valid use cases for using symbolic alleles are -- to my reading it's a little unclear whether it's saying that the two enumerated cases below (Structural Variants and IUPAC ambiguity codes) are the only permitted uses for symbolic ALT alleles or whether other, arbitrarily defined, symbolic alts are allowable. What about `<NON-REF>`, which is allowed but discouraged according to section 5.5? Are symbolic alleles that represent markers for arbitrary but incompletely specified haplotypes allowed (I've seen these used with germline statistical phasing tools such as Beagle; they might or might not include structural variants but wouldn't have an SVTYPE)? I'm actually unsure as to whether these types of symbolic alleles are already disallowed, but I think we should state it clearly here either way.

Reply:
I guess below you do say:

> Any symbolic alleles with a first level type other than one of those listed above are considered to be non-structural symbolic alleles.

Which indicates that non-structural symbolic alleles are allowed. But I still think it would be good to clarify their scope here in the header for the section as well.

Reply:
This is a very valid comment, though not strictly speaking related to SV improvements. I've created a separate issue https://github.com/samtools/hts-specs/issues/545 and will be marking this discussion as resolved.